tests/secret-providers/aws_secretmanager/aws_secretmanager_test.go

```diff
@@ -151,6 +152,31 @@ spec:
       name: {{.SecretManagerSecretName}}
 `
 
+	triggerAuthenticationSecretKeyTemplate = `apiVersion: keda.sh/v1alpha1
+kind: TriggerAuthentication
+metadata:
+  name: {{.TriggerAuthenticationName}}
+  namespace: {{.TestNamespace}}
+spec:
+  awsSecretManager:
+    credentials:
+      accessKey:
+        valueFrom:
+          secretKeyRef:
+            name: {{.AwsCredentialsSecretName}}
+            key: AWS_ACCESS_KEY_ID
+      accessSecretKey:
+        valueFrom:
+          secretKeyRef:
+            name: {{.AwsCredentialsSecretName}}
+            key: AWS_SECRET_ACCESS_KEY
+    region: {{.AwsRegion}}
+    secrets:
+    - parameter: connection
+      name: {{.SecretManagerSecretName}}
+	  secretKey: connectionString
```

Comment:
Be care with the indentation, the "tab" breaks the templating
```suggestion
      name: {{.SecretManagerSecretName}}
      secretKey: connectionString
```